lib/taurus/qt/qtgui/table/qlogtable.py

```diff
@@ -600,17 +601,18 @@ def main():
 
 
 @click.command('qlogmon')
-@click.option('--port', 'port', type=int,
-              default=logging.handlers.DEFAULT_TCP_LOGGING_PORT,
-              show_default=True,
-              help='port where log server is running')
-@click.option('--log-name', 'log_name', default=None,
-              help='filter specific log object')
-@click.option('--log-level', 'log_level',
-              type=click.Choice(['critical', 'error', 'warning', 'info',
-                                 'debug', 'trace']),
-              default='debug', show_default=True,
-              help='filter specific log level')
+@click.option(
+    '--port', 'port', type=int,
+    default=logging.handlers.DEFAULT_TCP_LOGGING_PORT,
+    show_default=True,
+    help='port where log server is running',
```

Comment:
minor: capitalize 1st letter of help

Reply:
Hi @stanislaw55 , you were too fast in fixing.. I couldn't even finish the review when it was already patched... ;-)  

Note that there are still a couple of capitalizations and a wrong window name missing  (I leave the conversations open to mark them)

I would have done all these minor changes myself, but I do not have write access to the PR branch